site/docs/master/installing.md

```diff
@@ -0,0 +1,78 @@
+# Installing Velero
```

Comment:
I'm finding this page to be confusing as-is, as it's kind of a random collection of information related to install that doesn't actually have any specific installation instructions. I think I'd prefer to either:
- move this content into the "Install Overview" page, as background (if we choose this, I bet some of the content could be removed to shorten it up as it's probably redundant with other spots)
- call this "Install Reference" or something other than just "Installing" (both the page name and in the TOC)
- reorganize this page so that it functions better as a main "installing" page

what do others think?

Reply:
To add more context here - since this page is called "Installing" in the TOC, I'd expect it to be the main page that has step-by-step instructions or well-organized links to other pages, but it's not really that at the moment, hence the ideas for different organization.

Reply:
Ok. I see your point and agree. I'm thinking about it.